db/migrations/4_pad_id.sql

```diff
@@ -0,0 +1,82 @@
+DO $$
+DECLARE
+    tbl_name text;
+    current_schema_name text;
+    col_name text;
+BEGIN
+    -- Get the current schema name
+    SELECT current_schema() INTO current_schema_name;
+
+    -- Loop through each table in the current schema
+    FOR tbl_name IN
+        SELECT table_name
+        FROM information_schema.tables
+        WHERE table_schema = current_schema_name AND table_type = 'BASE TABLE'
+    LOOP
+        -- Check if `created_block_id` column exists and update values
+        IF EXISTS (
+            SELECT 1
+            FROM information_schema.columns
+            WHERE table_schema = current_schema_name
```

Comment:
IMO we should re-sync all of these. Just updating only these value won't do the trick, we will have to make sure all the composite ID's of other entities (where ID for example is created as `1000/1` [`blockId/eventIdx`]) will need to be updated as well, to keep the data correct with/without sync.

Feels like a lot of pain to go and add queries for all such entities.